frontend/src/index.css

```diff
@@ -1,12 +1,41 @@
+/* google font */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;800&family=VT323&display=swap');
+
+:root {
+  --primary: #1aac83;
+  --error: #e7195a;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: "Poppins";
+  background: #f1f1f1;
```

Comment:
Mmm I don't really know.
Adding the colors to a config file is a bit confusing because CSS files are supposed to be style config files.
But I tried to put some CSS variable at the beginning of the file to avoid hard-coding the colors in the style file.

In index.css:
```css
:root {
  --primary: #1aac83;
  --secondary: #f1f1f1;
  --error: #e7195a;
}
```

But maybe I missed some of them!

Reply:
Ok fine for me!